src/pages/how-to-be-sponsors.tsx

```diff
@@ -4,10 +4,51 @@ import "bootstrap/dist/css/bootstrap.min.css";
 
 import BasePage from "../components/BasePage/BasePage";
 
+const title = "Become a Sponsor"
+
+const style = {
+  "width": "50%",
+  "margin-left": "auto",
+  "margin-right": "auto",
+}
+
+const bronze = {
+  "color": "#cd7f32",
```

Comment:
You could create a `colors.module.css` file for these values to live, and then import them here

Reply:
Refactored here: [955bf17](https://github.com/OHSBlazerbots/blazerbots/pull/15/commits/955bf17c9b7ac5505628715ac156b8ec8e68ccba). I ended using a TypeScript file, rather than CSS module because it'll be more flexible for a consumer to use the color however is needed (as background, as foreground, etc.). Plus, in the future I'll be easier to have a data structure for all sponsor data.